src/main/java/net/sf/jabref/logic/util/strings/RtfCharMap.java

```diff
@@ -49,79 +49,58 @@ public RtfCharMap () {
         // Use UNICODE characters for RTF-Chars which can not be found in the
         // standard codepage
 
-        put("`A", "\\u192A"); // "Agrave"
```

Comment:
Some keys got 2 values assigned, so I removed the double assignement. If a key got an RTF value I removed the unicode value.